instrumentation-core/spring/spring-webflux-5.0/src/main/java/io/opentelemetry/instrumentation/spring/webflux/client/SpringWebfluxHttpClientTracer.java

```diff
@@ -49,6 +54,12 @@ protected URI url(ClientRequest httpRequest) {
 
   @Override
   protected Integer status(ClientResponse httpResponse) {
+    if (null != RAW_STATUS_CODE) {
```

Comment:
```suggestion
    if (RAW_STATUS_CODE != null) {
```

Reply:
If this bothers you, I suggest adding a findbugs/checkstyle rule for it.